README.md

```diff
@@ -90,6 +95,8 @@ Example:
     $images = Get-WimFileImagesInfo -WimFilePath $wimFilePath
 
     # Select the first one
+    # [0] - Core Installation
```

Comment:
Pls also add something like: if you want to see available installation images, you may print the ```$images``` variable